src/Sylius/Bundle/AdminBundle/Action/Account/PostRequestPasswordResetAction.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Action\Account;
+
+use Sylius\Bundle\AdminBundle\Form\Model\PasswordResetRequest;
+use Sylius\Bundle\AdminBundle\Form\RequestPasswordResetType;
+use Sylius\Bundle\CoreBundle\Message\Admin\Account\RequestResetPasswordEmail;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+final class PostRequestPasswordResetAction
```

Comment:
```suggestion
final class AdminResetPasswordAction
```
I feel suffix with the type of operation is redundant, but WDYT?

Reply:
You mean... `*Action`? 😄 this is the convention we have in all one-action controllers

Reply:
@Zales0123 I think he meant `Post` part but I think it's not redundant as the same route has separate action for `Get` and `Post` action so it was easier to put method name than inventing a new name less related to the thing it's about.